src/prefect/tasks/mysql/mysql.py

```diff
@@ -141,34 +178,60 @@ def __init__(
         self.commit = commit
         self.charset = charset
         self.cursor_type = cursor_type
+        self.ssl = ssl
         super().__init__(**kwargs)
 
     @defaults_from_attrs(
-        "fetch", "fetch_count", "query", "commit", "charset", "cursor_type"
+        "db_name",
+        "user",
+        "password",
+        "host",
+        "port",
+        "fetch",
+        "fetch_count",
+        "query",
+        "commit",
+        "charset",
+        "cursor_type",
+        "ssl",
     )
     def run(
         self,
-        query: str,
+        db_name: str = None,
+        user: str = None,
+        password: str = None,
+        host: str = None,
+        port: int = 3306,
         fetch: str = "one",
         fetch_count: int = 10,
+        query: str = None,
         commit: bool = False,
         charset: str = "utf8mb4",
         cursor_type: Union[str, Callable] = "cursor",
```

Comment:
Same as above, can you change these defaults to `None` since the default value is defined at init time?